templates/common/infra/bicep/core/host/aks-managed-cluster.bicep

```diff
@@ -61,15 +61,15 @@ param systemPoolConfig object
 @description('The DNS prefix to associate with the AKS cluster')
 param dnsPrefix string = ''
 
-resource aks 'Microsoft.ContainerService/managedClusters@2022-11-02-preview' = {
+resource aks 'Microsoft.ContainerService/managedClusters@2023-01-02-preview' = {
   name: name
   location: location
   tags: tags
   identity: {
     type: 'SystemAssigned'
   }
   sku: {
-    name: 'Basic'
+    name: 'Base'
```

Comment:
Should this be a param

Reply:
There is only 1 version.  Changed from `Basic` to `Base`.